upgrade_test.py

```diff
@@ -326,6 +326,10 @@ def _upgrade_node(self, node, upgrade_sstables=True, new_scylla_repo=None, new_v
                         f' -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold"'
                     )
                     InfoEvent(message='upgrade_node - ended to "apt-get update"').publish()
+
+        InfoEvent(message='upgrade_node - fix /etc/scylla.d/io.conf arguments compatibility').publish()
```

Comment:
you mean arguments compatibility between 2021.1 and 2022?
btw. Is this in upgrade docs?

Reply:
it's not in docs, and it's when upgrading direct between 2022.1 and 2024.1.

I'm not playing by the book, for some specific reasons.

this flag have deprecation warning in 2022.1 and in 2023.1, but really deprecated in 2024.1

Reply:
In 2024.1 it was `removed`.
`Deprecated` it was since `2022.1` till the `2023.1`.
And in `2021.1` it was actual not having new renamed variant.